src/tests/test-profile/meson.build

```diff
@@ -0,0 +1,8 @@
+if build_standalone
```

Comment:
Maybe it's better to store these at https://github.com/fwupd/fwupd-test-firmware so they don't end up packaged and more samples can be added out of band

Reply:
My initial thought is that's a good idea. I'm then worried about the implied "API" we have then; i.e. if we change attr AppStreamIDs or HSI levels.

Reply:
See my other comment; unit tests.  Then we can make sure that calculated value matches the given fwupd version.

Reply:
Lets do this as phase 2 -- what I'm not convinced about is the *name* -- profile is what we use for "how long it takes" and so we need another name I think. Maybe `emulate`? i.e. `FWUPD_DEVICE_EMULATE` or `FWUPD_EMULATE` then the json files could be in `/usr/share/fwupd/emulate.d` -- other ideas welcome.

Reply:
FWUPD_HOST_EMULATE might work. or FWUPD_EMULATE_HOST

Reply:
we've got FWUPD_SOLOKEY_EMULATE so maybe the former?

Reply:
How about `FWUPD_HSI_EMULATE`

Reply:
BTW I think you need to make sure the daemon can't upload data while it's emulating and also there should be a flag in place to say data is emulated that GUI or command line clients can use to warn it's under emulation.

Reply:
I added something for that, take a look.  The other option is to taint the daemon.  I can see arguments both ways.

Reply:
Ya, we need the flag, and we'll need a fwupd_device flag for the same purposes too. Thanks.

Reply:
I did a new attribute; I think that's semantically most pleasing, and also means the attribute flags are the same for emulation and non-emulation.